src/auth/Readme.md

```diff
@@ -0,0 +1,266 @@
+# Authentication flow
```

Comment:
The whole `src/auth` directory was copied from react-storefront. In the future it'll become a standalone library.

Reply:
Can you please change the name to `README`?